Manage.c

```diff
@@ -1655,7 +1658,6 @@ int Manage_subdevs(char *devname, int fd,
 					force, verbose, devname, update,
 					rdev, array_size, raid_slot);
 			close(tfd);
-			tfd = -1;
```

Comment:
Not I understand your point here? The close_fd(tfd) will set it to -1.

Reply:
I think that you received error because, at the beginning we have:
```int tfd = -1;```
and later  we have:
```tfd = dev_open(...)```

I think that the initial assignment may cause the error. We are using close_fd() in many places and SAST never complained that assignment after close() is not mandatory. If it is complaining, I would prefer to invalidate file descriptor and ignore this error because it is safer.